desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -15,11 +15,82 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+import json
+from pprint import pprint
+
+from desktop import conf
 from django.shortcuts import render
 from django.http import HttpResponse
-from desktop.lib.django_util import login_notrequired
+from desktop.lib.django_util import login_notrequired, JsonResponse
+from desktop.lib.exceptions_renderable import PopupException
+from django.utils.translation import ugettext as _
+from django.views.decorators.csrf import csrf_exempt
+
+from django.conf import settings
+from slack_sdk import WebClient
```

Comment:
We should only load the module and instantiate it below when the enable is on (which we do only for Py3)
Could also catch the import error in some place, ideally just using the flag is a bit cleaner.

Reply:
Used the flag for enabling it ( which is done for py3). Also enabling the UTs with the flag @romainr